boa/src/syntax/parser/statement/break_statement/mod.rs

```diff
@@ -0,0 +1,69 @@
+#[cfg(test)]
```

Comment:
Missing documentation.
```suggestion
//! Break expression parsing.
//!
//! More information:
//!  - [MDN documentation][mdn]
//!  - [ECMAScript specification][spec]
//!
//! [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/break
//! [spec]: https://tc39.es/ecma262/#prod-BreakStatement

#[cfg(test)]
```